src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/KeyAwareChildEntity.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.induction
+
+/**
+ * Interface for "child" entity classes that can provide a key to help identify them, or potentially to sort them
+ * within Collections.
+ */
+interface KeyAwareChildEntity {
```

Comment:
I initially had a `ChildEntity` interface, which extended the original `EntityKeyAware` one, but I think this is clearer (especially where it's used).